app/src/main/res/values/strings.xml

```diff
@@ -107,6 +107,9 @@
 
     <!-- ScrollingActivity stuff -->
     <string name="openAdCreationDescription_Scrolling">Open page for creating an ad</string>
+    <string name="searchDescription_Scrolling">Where are you going?</string>
```

Comment:
Maybe a phrase like "Which city?" or "Search by city" would be clearer for the user.

Reply:
Go for "Which city?"!